scripts/migrations/2024_06_save_org_product_data_daily.pl

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/perl -w
```

Comment:
the /migrations/ folder is only for scripts that we are running once (e.g. to update the format of the product data). Could you remove the date from the file name, and move it outside of the migrations/ folder?